lib_nbgl/include/nbgl_use_case.h

```diff
@@ -137,10 +137,67 @@ typedef struct {
     uint8_t nbContents;  ///< number of contents
 } nbgl_genericContents_t;
 
+/**
+ * @brief The different types of operation to review
+ *
+ */
+typedef enum {
+    TYPE_OPERATION = 0,
+    TYPE_TRANSACTION,
+    TYPE_MESSAGE,
+} nbgl_operationType_t;
+
+/**
+ * @brief The different types of review status
+ *
+ */
+typedef enum {
+    STATUS_TYPE_OPERATION_SIGNED = 0,
+    STATUS_TYPE_OPERATION_REJECTED,
+    STATUS_TYPE_TRANSACTION_SIGNED,
+    STATUS_TYPE_TRANSACTION_REJECTED,
+    STATUS_TYPE_MESSAGE_SIGNED,
+    STATUS_TYPE_MESSAGE_REJECTED,
+    STATUS_TYPE_ADDRESS_VERIFIED,
+    STATUS_TYPE_ADDRESS_REJECTED,
+} nbgl_reviewStatusType_t;
+
 /**********************
  * GLOBAL PROTOTYPES
  **********************/
 
+void nbgl_useCaseHomeAndSettings(
+    const char                *appName,
+    const nbgl_icon_details_t *appIcon,
+    const char                *tagline,
+    const uint8_t *initSettingPage,  // Set to NULL if settings shouldn't be displayed right away
+    const nbgl_genericContents_t *settingContents,
+    const nbgl_contentInfoList_t *infosList,
+    const char                   *actionText,      // Set to NULL if no additional action
+    nbgl_callback_t               actionCallback,  // Set to NULL if no additional action
+    nbgl_callback_t               quitCallback);
+
+void nbgl_useCaseReview(nbgl_operationType_t             operationType,
+                        const nbgl_layoutTagValueList_t *tagValueList,
+                        const nbgl_icon_details_t       *icon,
+                        const char                      *reviewTitle,
+                        const char *reviewSubTitle, /* Most often this is empty, but sometimes
+                                                       indicates a path / index */
+                        const char           *finishTitle, /* unused on Nano */
+                        nbgl_choiceCallback_t choiceCallback);
+
+void nbgl_useCaseAddressReview(
+    const char *address,
+    const nbgl_layoutTagValueList_t
+        *additionalTagValueList,  // Set to NULL if there are no additional info to display
+    const nbgl_icon_details_t *icon,
+    const char                *reviewTitle,
+    const char                *reviewSubTitle,
+    nbgl_choiceCallback_t      choiceCallback);
+
```

Comment:
What is the difference with existing use case `nbgl_useCaseAddressConfirmation` and `nbgl_useCaseAddressConfirmationExt` ?

Reply:
It also bundles the nbgl_useCaseReviewStart, you can see it live in the boilerplate Pr from this Pr description

Reply:
The bundle allow to go back to the nbgl_useCaseReviewStart screen from the following screens, which is now requested but is not possible from the previous API